apps/mi/mi-adf-shir/ithc.yaml

```diff
@@ -12,3 +12,7 @@ spec:
     subscriptionId: "ba71a911-e0d6-4776-a1a6-079af1df7139"
     tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
     managedIdentityClientId: "9106f8ad-3189-4053-925d-0d4579c4e0d9"
+    hostAliases:
+      - ip: 51.141.130.100
```

Comment:
This matches the ip used in dev, is this right?

Reply:
yes, ithc is technically a nonprod environment, only stg and prod are actual production network envs